test/Interop/Cxx/static/Inputs/constexpr-static-member-var-errors.h

```diff
@@ -0,0 +1,6 @@
+template <class T>
+struct IsSubtypeSame {
+  static constexpr const bool value = __is_same(typename T::type, int);
```

Comment:
It would be nice if the test could avoid using a built-in directly. Instead you can do something like (for example):

```
struct S {};
template <class T>
 struct IsSubtypeSame {
   static constexpr const S value = T();
};
using Invalid = IsSubtypeSame<int>;
```

Reply:
Hey, you got it to crash in a different way! 

Anyway, I take your point. I'll try to re-write this test without using the builtin (and I'll submit another patch to fix the issue you exposed).